bots/flakes-refresh

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
```

Comment:
Harmless, but FYI not required for Python 3.

Reply:
Fixed.